proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -3039,6 +3058,7 @@ lemma sts_sch_act':
   apply (wpsimp wp: threadSet_pred_tcb_at_state threadSet_sch_act_wf hoare_vcg_disj_lift)
   done
 
+(* FIXME: sts_sch_act' (above) is stronger, and should be the wp rule. VER-1366 *)
```

Comment:
Is this MCS specific? If so, can you label it with FIXME RT?

Reply:
I don't think this is MCS-specific - this is just a weak rule incorrectly declared `[wp]`.